ext/v8/integer.h

```diff
@@ -0,0 +1,47 @@
+// -*- mode: c++ -*-
+#ifndef RR_INTEGER_H
+#define RR_INTEGER_H
+
+namespace rr {
+  class Integer : public Ref<v8::Integer> {
+  public:
+    Integer(v8::Isolate* isolate, v8::Handle<v8::Integer> integer) :
+      Ref<v8::Integer>(isolate, integer) {}
+    Integer(VALUE self) :
+      Ref<v8::Integer>(self) {}
+
+    static VALUE New(VALUE self, VALUE r_isolate, VALUE value) {
+      Isolate isolate(r_isolate);
+      Locker lock(isolate);
+
+      v8::Local<v8::Integer> i = v8::Integer::New(isolate, NUM2INT(value));
+      return Value::handleToRubyObject(isolate, i);
```

Comment:
Actually, we can just `return Integer(isolate, i);` here since this is what is actually given to us. Currently `handleToRubyObject` doesn't return `Integer` anywhere.
Also, the `Value` method will work either way. I suspect it only returns `Int32` or `Uint32` depending on the call and `Integer` is used just as a common type.